docs/baseband/mark4/index.rst

```diff
@@ -4,48 +4,88 @@
 MARK 4
 ******
 
-The Mark 4 VLBI format is described in the `Mark IIIA/IV/VLBA documentation
+The Mark 4 format is the output format of the MIT Haystack Observatory's Mark 4
+VLBI magnetic tape-based data acquisition system, and one output format of its
+successor, the Mark 5A hard drive-based system. The format's specification is
+in the `Mark IIIA/IV/VLBA documentation
 <http://www.haystack.mit.edu/tech/vlbi/mark5/docs/230.3.pdf>`_.
 
+Baseband currently only supports files that have been parity-stripped and
+corrected for barrel roll and data modulation.
+
+.. _mark4_file_structure:
+
+File Structure
+==============
+
+Mark 4 files contain up to 64 concurrent "data tracks", which
+are :term:`streams<stream>` of :term:`elementary sample`.  Tracks are divided
+into 22500-bit "tape frames", each of which consists of a 160-bit
+:term:`header` followed by a 19840-bit :term:`payload`.  The header includes a
+timestamp (accurate to 1.25 ms), track ID, sideband, and fan-out/in factor
+details (see below); the details of these can be found in 2.1.1 - 2.1.3 in the
+specification document.  The payload consists of 1-bit elementary samples. 
```

Comment:
Make this a link as well.

Reply:
Also, I don't understand the "payload consists of 1-bit elementary samples." just delete it?

Reply:
I had originally thought it necessary to define an "atomic" or "elementary" sample, from which "component" samples are built - eg. for VDIF the real component of one sample, or for Mark 4 with fan-out one sample in one track.  I thought that this would let me describe GSB more easily, but it turned out not to be the case.  Perhaps we should go over the glossary together in the meeting tomorrow, and make concrete what we mean by "sample" and how to distinguish between a sample from one channel and a complete sample.